packages/@sanity/cli/src/actions/typegen/generate.telemetry.ts

```diff
@@ -10,6 +10,9 @@ interface TypesGeneratedTraceAttrubutes {
   unknownTypeNodesGenerated: number
   unknownTypeNodesRatio: number
   emptyUnionTypeNodesGenerated: number
+  config: {
+    overloadClientMethods: boolean
+  }
```

Comment:
Event properties should ideally not be nested. Is it OK to flatten the object by removing the `config` property and moving `overloadClientMethods` out?

Reply:
Sure. I made it nested since that's how it is here https://github.com/sanity-io/sanity/blob/next/packages/%40sanity/cli/src/__telemetry__/cli.telemetry.ts#L18-L22

Reply:
But can update